.github/workflows/helm-version-update.yaml

```
@@ -0,0 +1,39 @@
+name: Build
+on:
+  schedule:
+    - cron: '0 8 * * 1'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  HELM_VERSION: v3.11.1
+
+jobs:
+  ensure-versions:
+    runs-on: ubuntu-22.04
+    permissions:
+      pull-request: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.5.0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3.5
+        with:
+          version: ${{ env.HELM_VERSION }}
+
+      - name: Update Versions
+        run: |
+          ./.github/workflow/scripts/update-versions.sh
+          git diff
+
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+
+      - name: Check outputs
+        if: ${{ steps.cpr.outputs.pull-request-number }}
+        run: |
+          echo "Pull Request Number - ${{ steps.cpr.outputs.pull-request-number }}"
+          echo "Pull Request URL - ${{ steps.cpr.outputs.pull-request-url }}"
```

```suggestion
          echo "::notice title=PR #${{ steps.cpr.outputs.pull-request-number }}::${{ steps.cpr.outputs.pull-request-url }}"
```

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

This will nicely render a notice on the Summary page of the workflow run.